x-pack/solutions/security/plugins/security_solution/common/siem_migrations/dashboards/resources/resource_identifier.ts

```diff
@@ -19,6 +19,11 @@ export class DashboardResourceIdentifier extends ResourceIdentifier<OriginalDash
     }
     const splunkDashboardXMLPaser = new SplunkXmlDashboardParser(originalDashboardXMLString);
     const queries: string[] = await splunkDashboardXMLPaser.extractQueries();
-    return queries.flatMap((query) => this.identifier(query));
+    const resources: SiemMigrationResourceBase[] = [];
```

Comment:
no reason to do these in order right? so can't we just do:

```
const results = await Promise.all(queries.map(query => this.identifier(query)));
const resources = results.flat();
return resources;
```
for 1 await only?

Reply:
done in 289e85ed252f